.codecov.yml

```diff
@@ -0,0 +1,4 @@
+ignore:
+  - "codegen-lite/test/resources/**/*"
+  - "codegen/src/test/resources/**/*"
+  - "test/**/*"
```

Comment:
Extra character at the end

Reply:
Hmm, not sure whey it showed up like that, but there's no extra character in this file though

Reply:
Any reason why we are just ignoring these folders? Are there others that need to be excluded?

Reply:
This is what we discussed this morning, excluding the test folders for the test coverage. Double checked and those are the only folders we want to ignore.

Reply:
Oh, I misunderstood your question. Yeah, those folders are the ones showing on codecov page that should not be counted.

![screen shot 2019-03-01 at 3 36 31 pm](https://user-images.githubusercontent.com/33073555/53672547-e2f3b100-3c37-11e9-8f7f-ad3002a93304.png)